libraries/stdlib/samples/test/samples/text/strings.kt

```diff
@@ -408,4 +408,17 @@ class Strings {
         assertPrints(matchDetails(inputString, toFind, 2), "Searching for 'ever' in 'Never ever give up' starting at position 2: Found at 6")
         assertPrints(matchDetails(inputString, toFind, 10), "Searching for 'ever' in 'Never ever give up' starting at position 10: Not found")
     }
+
+    @Sample
+    fun last() {
+        val string = "Kotlin 1.4.0"
+        assertPrints(string.last(), "0")
+        assertPrints(string.last { it.isLetter() }, "n")
+        assertFails { string.last { it > 'z' } }
+        assertPrints(string.lastOrNull { it > 'z' }, "null")
```

Comment:
Better to swap `assertFais` and `assertPrints`, because the former will be commented out.